grpc/tls/provider/meshca/v1/service.proto

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package grpc.tls.provider.meshca.v1;
+
+import "google/protobuf/duration.proto";
+
+option java_multiple_files = true;
+option java_outer_classname = "MeshCaProto";
+option java_package = "io.grpc.tls.provider.meshca.v1";
+
+// The Mesh CA provides certificates for GCP users and service accounts. The CA
+// will exchange an account credential, such as a service account key or OAuth
+// credential, for an X.509 certificate. The subject of the signed certificate
+// will match the identity of the authenticated requestor. The format of the
+// identity in the certificate is a SPIFFE SVID:
+//   spiffe://<identity namspace>/ns/<istio namespace>/sa/<kubernetes service
```

Comment:
suggestion: replace istio with kubernetes (not super-accurate but close enough), or ASM (more accurate but perhaps too coupled?)

Reply:
If the service account is "kubernetes service account" shouldn't the namespace also be "kubernetes"?

Reply:
Removed istio, and made it simply `namespace`.

@sanjaypujare : There is no `kubertenes` namespace. There is a `default` namespace, but users can create their own namespaces.

Reply:
I didn't mean literally `kubernetes` - what I meant is that namespace is a kubernetes construct (and not Istio AFAICT).

Reply:
You are right. The namespace should also be a kubernetes construct. Although, I do see some documentation about Istio namespaces, but I think we will be better off without mentioning Istio here.

Reply:
I believe the Istio namespace is an abstraction that is usually a K8S namespace (if used in the context of K8S), but can be separate if used outside the context of K8S. Having said that, since we don't assume Istio is installed here, and presumably the alpha release is only for K8S, we can probably just keep the "Kubernetes" label here.

If you plan on adding non-K8S support very soon, it might be worth using a more generic name here.

Reply:
@pmansour I realize this wording was taken from an internal proto definition that came from the MeshCA team. Since it's really the MeshCA that infers these values can you suggest a 
 more generic name?